runtime/v2/shim/shim_test.go

```diff
@@ -18,9 +18,30 @@ package shim
 
 import (
 	"context"
+	"os"
+	"runtime"
 	"testing"
 )
 
+func TestRuntimeWithEmptyMaxEnvProcs(t *testing.T) {
+
+	os.Setenv("GOMAXPROCS", "")
+	setRuntime()
+	var oldGoMaxProcs = runtime.GOMAXPROCS(0)
```

Comment:
What does "old" mean here?

Reply:
`GOMAXPROCS` returns a previous setting.  In this case old = previous. But I think `previous` is more relevant  than `old`

Reply:
confused, it is called after setRuntime

Reply:
1/  `setRuntime()`  --> GOMAXPROCS = 2 
2/   oldGoMaxProcs = runtime.GOMAXPROCS(0) --> GOMAXPROCS = 0 and oldGoMaxProcs = 2

Reply:
@AkihiroSuda is it clear my strategy ?